src/rimestate.cpp

```diff
@@ -440,6 +441,30 @@ void RimeState::updateUI(InputContext *ic, bool keyRelease) {
 
 void RimeState::release() { session_.reset(); }
 
+void RimeState::commitInput(InputContext *ic) {
+    if (auto *api = engine_->api()) {
+        if (auto input = api->get_input(this->session())) {
+            if (std::strlen(input) > 0) {
```

Comment:
get_input may return null, but strlen don't accept nullptr.

Reply:
the previous line (L446) guarantees this